### PR TITLE
ZDT4 Evaluate Fix

### PR DESCRIPTION
**Pull request by jonpsy**
Minor fix.

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -37,6 +37,9 @@
  * Introduce Policy Methods for MOEA/D-DE
    ([#293](https://github.com/mlpack/ensmallen/pull/293)).
 
+ * ZDT4 Evaluate Fix
+   ([#298](https://github.com/mlpack/ensmallen/pull/298)).
```
Comment (zoq) [inline HISTORY.md:41]: ZDT wasn't part of a release, so no need to include the PR in the HISTORY.

Comment (coatless) [reply HISTORY.md:41]: @zoq I think he's trying to avoid running into having the CI pick up an error if HISTORY wasn't modified.

Comment (jonpsy) [reply HISTORY.md:41]: Both of you are correct. In either case, this change isn't anything significant, so we can ignore HISTORY.

Comment (zoq) [reply HISTORY.md]: Right, happy to ignore the failing build in this case, in favor of a cleaner HISTORY.

**Review by zoq — APPROVED** [review]
Looks good to me.

**Comment by jonpsy** [comment]
Let's hold it for a while, I'm getting non-sensical values again. Not sure if algorithm is wrong or test suite or the code I wrote. But, the values are non-sensical for both NSGA-II and MOEAD, so I have a good reason to believe this algo could be wrong.

**Comment by jonpsy** [comment]
Oh! It turned out the bounds in variable space for the problem is strictly [0, 1]. In my testing I used [-1 , 1] that caused ```arma::datum::nan``` values :D.

**Comment by jonpsy** [comment]
Have this merged? @zoq

**Comment by zoq** [comment]
Sure, we can merge this one, was waiting because you wanted to hold it for a while.

**Comment by jonpsy** [comment]
Actually I was holding #299, not this one. This one's ready.

**Review by coatless — APPROVED** [review]

**Comment by zoq** [comment]
Thanks :+1:

**Pull request merged**